modules/ROOT/pages/_partials/sync/ref_terminology.adoc

```diff
@@ -1,14 +1,16 @@
-//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-[id='ref_terminology_{context}']
+[id='ref-terminology-{context}']
 
 = {sync-service} Terminology
 
-This section describes terminology that is associated with {sync-service}.
+This section describes terminology that is associated with Data Sync.
 
-GraphQL:: A query language for your API, and a server-side runtime for executing queries by using a type system you define for your data. An alternative approach to REST for building APIs and applications. See link:https://graphql.org/learn[GraphQL].
+Data Sync framework - Data Sync is a JavaScript framework that enables a developer to add real time data synchronization to both mobile and web clients.
+The Data Sync framework comprises of two components, the _Sync Server_ and the _Sync Client_.
+
+Sync Server - The Sync Server is a framework for building Node.js based GraphQL API.
 
-Apollo:: Apollo is an implementation of GraphQL designed for the needs of product engineering teams building modern, data-driven applications. Apollo includes two open-source libraries, Apollo Server and Apollo Client. See link:https://www.apollographql.com[Apollo].
+Sync Client - The Sync Client is a JavaScript client offering client side extensions and server side integration. The Sync Client can be integrated into frameworks such as React and Angular.
```

Comment:
Do we need to say what sort of extensions and integrations?

Reply:
Hi @wei-lee - it would be great to provide the reader with examples for (1) extensions and (2) server-side integration - can you advise on this?

@pnagesha - do you think server side should be "server-side" too?